packages/bindings/src/types.rs

```diff
@@ -3,7 +3,7 @@ use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
 /// Supported provenance module router keys.
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[derive(Serialize, Deserialize, Clone, Debug, Eq, PartialEq, JsonSchema)]
```

Comment:
Same comments as attrs/msg.rs.

Reply:
the bindings package actually remains unchanged. only inputs/outputs of the contracts use the new derived schema